<def f='llvm/llvm/unittests/Support/ErrorTest.cpp' l='247' macro='1' type='void (anonymous namespace)::Error_HandleCustomErrorWithCustomBaseClass_Test::Error_HandleCustomErrorWithCustomBaseClass_Test()'/>
